asv_bench/benchmarks/boolean.py

```diff
@@ -0,0 +1,32 @@
+import numpy as np
+
+import pandas as pd
+
+
```

Comment:
do we have others that could be moved here? (e.g. the boolean Series ops)

Reply:
I don't see any in `benchmarks/binary_ops.py` or `benchmarks/series_methods.py`, though I may be missing some.